rpcd/playbooks/roles/rpc_maas/templates/galera_check.yaml.j2

```diff
@@ -34,3 +34,57 @@ alarms      :
             }
+    open_file_size_limit_reached :
+        label                   : open_file_size_limit_reached--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["num_of_open_files"] > metric["open_files_limit"]) {
+                return new AlarmStatus(CRITICAL, "Mysql open files has exceeded open files limit");
+            }
+    innodb_row_lock_time_avg :
+        label                   : innodb_row_lock_time_avg--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["innodb_row_lock_time_avg"] > {{ innodb_row_lock_time_avg_critical_threshold }}) {
+                return new AlarmStatus(CRITICAL, "InnoDB row lock time average has exceeded {{ innodb_row_lock_time_avg_critical_threshold }}");
+            }
+            if (metric["innodb_row_lock_time_avg"] > {{ innodb_row_lock_time_avg_warning_threshold }}) {
+                return new AlarmStatus(WARNING, "InnoDB row lock time average has exceeded {{ innodb_row_lock_time_avg_warning_threshold }}");
+            }
+    innodb_deadlocks :
+        label                   : innodb_deadlocks--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["innodb_deadlocks"] != 0) {
+                return new AlarmStatus(CRITICAL, "InnoDB has experienced a deadlock");
+            }
+    access_denied_errors :
+        label                   : access_denied_errors--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (rate(metric["access_denied_errors"]) > {{ mysql_access_denied_errors_rate_warning_threshold }}) {
+                return new AlarmStatus(WARNING, "Access denied errors rate is greater than {{ mysql_access_denied_errors_rate_warning_threshold }}");
+            }
+            if (rate(metric["access_denied_errors"]) > {{ mysql_access_denied_errors_rate_critical_threshold }}) {
+                return new AlarmStatus(CRITICAL, "Access denied errors rate is greater than {{ mysql_access_denied_errors_rate_critical_threshold }}");
+            }
+    aborted_clients :
+        label                   : aborted_clients--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (rate(metric["aborted_clients"]) > 1) {
```

Comment:
I think we should make these variables.
1 Sounds fine, however if it is too much/too little a variable will allow support to edit this on the fly.

Additionally, I can't remember exactly how the rate is calculated, e.g. is it per second or is it based on a change since last poll (which would mean its per minute) - which might be worth looking into!

(Same for the below check)